HISTORY.md

```diff
@@ -3,6 +3,8 @@
 ### Public API Change
 * DeleteRange now returns `Status::InvalidArgument` if the range's end key comes before its start key according to the user comparator. Previously the behavior was undefined.
 * ldb now uses options.force_consistency_checks = true by default and "--disable_consistency_checks" is added to disable it.
+* Allowing compaction filter on flushing, picked from [facebook/rocksdb#pr8243](https://github.com/facebook/rocksdb/pull/8243).
```

Comment:
Why is this moved here😂, please double check the original PR.

Reply:
Sorry I think I misunderstood that. I fixed it referring to the origin PR. 😭